src/core/exec-invoke.c

```diff
@@ -4411,15 +4413,14 @@ int exec_invoke(
         }
 
         if (context->cpu_sched_set) {
-                struct sched_param param = {
+                struct sched_attr attr = {
+                        .size = sizeof(attr),
+                        .sched_policy = context->cpu_sched_policy,
                         .sched_priority = context->cpu_sched_priority,
+                        .sched_flags = context->cpu_sched_reset_on_fork ? SCHED_FLAG_RESET_ON_FORK : 0,
                 };
 
-                r = sched_setscheduler(0,
-                                       context->cpu_sched_policy |
-                                       (context->cpu_sched_reset_on_fork ?
-                                        SCHED_RESET_ON_FORK : 0),
-                                       &param);
+                r = sched_setattr(0, &attr, 0);
```

Comment:
as per coding style please suffix the "opaque" args with argument names in a comment, i.e. the ones one has no idea what this is, i.e. the first and last here. i.e.:

```c
r = sched_setattr(/* pid= */ 0, &attr, /* flags= */ 0);
```

Reply:
Done